platform.md

```diff
@@ -179,3 +193,15 @@ group = [
 Where:
 
 - Both `id` and `version` MUST be present for each buildpack object in a group.
+
+### project-metadata.toml (TOML)
+
+```toml
+[source]
+type = "<source type>"
+
+[source.version]
+
```

Comment:
Do these fields have arbitrary contents?

Reply:
Yes. I just added some detail to the schema. Let me know if the wording is clear enough